pkg/api/testing/serialization_test.go

```diff
@@ -19,7 +19,7 @@ package testing
 import (
 	"bytes"
 	"encoding/hex"
-	"encoding/json"
+	gojson "encoding/json"
```

Comment:
we use this naming to distinguish between Go's json and our json in some other places too